app/src/main/AndroidManifest.xml

```diff
@@ -74,15 +73,46 @@
 
         <activity
             android:name=".activity.BrowserActivity"
-            android:theme="@style/FullScreen">
+            android:theme="@style/FullScreen"
+            android:label="">
             <intent-filter>
                 <action android:name="org.openobservatory.ooniprobe.OPEN_BROWSER" />
-
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
-        <activity android:name=".activity.NotificationDialog"
+        <activity
+            android:name=".activity.NotificationDialog"
             android:theme="@style/Theme.Transparent" />
+        <activity
+            android:name=".activity.RunTestActivity"
+            android:label="@string/app_name"
+            android:launchMode="singleInstance">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
```

Comment:
So, here I understand that `BROWSABLE` category is the one set when the user clicks a link in the browser and you have registered as handler for such link, correct? I wonder in which case we receive intents with no category at all is set or have you added that category just as a catch-all and just in case?

Reply:
^^^ @lorenzoPrimi

Reply:
Don't know, just copied from Android manual.